lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -50,19 +50,19 @@
     default: "/"
   cert_chain:
     description:
-      - The CA certificate chain in PEM encoded format.
-      - Note that prior to 2.4, this parameter expected a path to a file.
-        Since 2.4 this is now accomplished using a lookup plugin. See examples for detail.
+      - The path to the CA certificate chain in PEM encoded format.
+        As of 2.4 this can also be accomplished using a lookup plugin.
```

Comment:
Thanks! I'm modified it slightly because it's not clear which to use for versions prior to 2.4